docs/examples/flow_agent_app/simple_loan_demo.py

```
@@ -0,0 +1,134 @@
+        for gw_id, gw_config in gateways_config.items():
+            gw_type = gw_config.get('type', 'N/A')
+            print(f"    - {gw_id}: {gw_type}")
+
+        # Show initial variables
+        initial_vars = flow_config.get_initial_variables()
+        print(f"  Initial Variables: {len(initial_vars)}")
+        for var_name, var_value in initial_vars.items():
+            print(f"    - {var_name}: {var_value}")
+        print()
+
+    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)